### ISSUE TITLE
Prevent Status Indicator Prints?

### ISSUE THREAD

**Issue by pehbehbeh** [dg/ftp-deployment#79]
Is there a option/flag to prevent the status indicator prints? I'm running this script via GitLab CI and my deployment log looks something like this:

<img width="732" alt="deployment log" src="https://cloud.githubusercontent.com/assets/133831/13682688/2c29f45c-e703-11e5-834e-2f07d44eca77.png">

I think you get the idea. The log gets very long and unreadable because of this.

**Comment by pehbehbeh** [comment]
I went through the code a bit and found a responsible line in the `collectPaths` method of the `Deployer` class: https://github.com/dg/ftp-deployment/blob/4fb74497470a886fa8514ab8e03067df01465226/src/Deployment/Deployer.php#L324

```
echo str_pad(str_repeat('.', $counter++ % 40), 40), "\x0D";
```

But this is not the only echo statement of this kind in `Deployer.php`.

What do you think about a `--noecho` option or something similar? I could contribute via PR if you like the idea.

**Comment by dg** [comment]
IMHO `--no-progress` is better. Could you prepare pull request?

**Comment by dg** [comment]
closed by #80
